app/styles/app.css

```diff
@@ -36,9 +36,9 @@ html, body {
 .footer {
   font-weight: lighter;
   text-align: center;
-  vertical-align: middle;
   width: 100%;
-  position: fixed;
-  height: 40px;
+  bottom: 0;
+  margin-top: 200px;
+  margin-bottom: 40px;
```

Comment:
We're aligning it using both `bottom` and `margin-bottom`? 🤔

Reply:
Yes, `bottom` places it at the bottom of parent, while `margin-bottom` provides some space, or it is stuck (looks bad)

Reply:
We don't actually need ```bottom``` and ```margin-bottom```. I have set footer height to 40px so that it doesn't look stuck to bottom.